### PR TITLE
[Update] add job title row to contact page

### PR DESCRIPTION
**Pull request by hanyulo**

### CODE REVIEW

File: models/Contact.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var Contact = new keystone.List('Contact', {
 
 Contact.add({
 	name: { label: '作者姓名', type: String, required: true, index: true },
+  job_title: { label: '職稱', type: String },
```
Comment (nickhsine) [inline models/Contact.js]: correct indent

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "react-dom": "^15.2.0",
     "react-domify": "^0.2.4",
     "store-prototype": "^1.1.1",
-    "twreporter-keystone": "^0.5.30",
+    "twreporter-keystone": "^0.5.32",
```
Comment (nickhsine) [inline package.json]: since you use `^`, you don't have to change the patch number.

File: models/Contact.js (hunk 1 above)
Comment (hanyulo) [inline models/Contact.js]: fixed. 👍

**Comment by nickhsine** [comment]
👍 💯

**Pull request merged**